development/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM php:7.3-fpm-buster
+
+# Arguments defined in docker-compose.yml
+ARG user
+ARG uid
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
+    git \
+    curl \
+    libpng-dev \
+    libonig-dev \
+    libxml2-dev \
+    zip \
+    unzip
+
+# Clear cache
+RUN apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# Install PHP extensions
+RUN docker-php-ext-install pdo_mysql mbstring exif pcntl bcmath gd \
+    && pecl install xdebug redis \
+    && docker-php-ext-enable xdebug redis \
+    && docker-php-source delete
+
+# Get latest Composer
+COPY --from=composer:latest /usr/bin/composer /usr/bin/composer
+
+
+RUN apt-get -y install curl
+RUN curl -sL https://deb.nodesource.com/setup_12.x  | bash -
```

Comment:
Is there a reason for us to specify Node 12?

I know that the current docs and pipeline mention nothing about which node versions is expected. The current LTS version is 16. Version 12 will only be maintained for the next ~3 months.

Reply:
Installing node via: `RUN apk add --update nodejs npm`
current version is v16.13.2